public/app/percona/shared/components/Elements/FeatureLoader/FeatureLoader.service.ts

```diff
@@ -2,7 +2,7 @@ import { api } from 'app/percona/shared/helpers/api';
 
 const BASE_URL = `/v1/Settings`;
 
-export const IntegratedAlertingService = {
+export const FeatureLoaderService = {
```

Comment:
Do we really need this service or can it be replaced with the Settings one? Since the settings got moved to this project the service got too.

Reply:
Yep, good one. Just had to make a small change there to make it follow our usual service pattern 👍